ports/atmel-samd/boards/robohatmm1_m4/mpconfigboard.mk

```diff
@@ -0,0 +1,31 @@
+LD_FILE = boards/samd51x19-bootloader-external-flash-crystalless.ld
```

Comment:
I don't think you need a new linker file so please reuse the existing one.

We run most M4s crystal-less now I think. The crystalless linker script is only needed with CALIBRATE_CRYSTALLESS on M0 which stores crystal calibration into flash.

Reply:
No problem.  I would want to test this first though.
```
LD_FILE = boards/samd51x19-bootloader-external-flash.ld
```

Reply:
Tested - Done.  Going to remove the extra linker script